core/src/main/java/com/homeaway/streamplatform/streamregistry/StreamRegistryApplication.java

```diff
@@ -380,8 +380,8 @@ private void initAndRegisterHealthCheck(final StreamRegistryConfiguration config
         Preconditions.checkState(managedKStreams != null, "managedKStreams cannot be null.");
         Preconditions.checkState(streamResource != null, "streamResource cannot be null.");
 
-        CompactedEventStoreHealthCheck compactedEventStoreHealthCheck = new CompactedEventStoreHealthCheck(configuration, metricRegistry);
-        environment.healthChecks().register("compactedEventStoreHealthCheck", compactedEventStoreHealthCheck);
+        EventStoreConfigHealthCheck eventStoreConfigHealthCheck = new EventStoreConfigHealthCheck(configuration, metricRegistry);
+        environment.healthChecks().register("eventStoreConfigHealthCheck", eventStoreConfigHealthCheck);
```

Comment:
Adding `"HealthCheck"` seems redundant here. Could just say `"EventStoreConfig"`

Reply:
Isn't this not backwards compatible again? 

E.g. if someone depended on checking this health check name, then we're changing the name?

Reply:
We have not CUT a release yet. so, changing the name would not cause any backward compatible issue.

Reply:
agreed... we haven't cut for 0.5.0 so just add it to the changed part in the CHANGELOG
(two non backward compatible changes)